src/pages/workspaces/workspace/jobHistory/CallCacheWizard.js

```diff
@@ -1,7 +1,7 @@
+import ReactJson from '@microlink/react-json-view';
 import _ from 'lodash/fp';
```

Comment:
I wasn't able to get into a state where I could use the CallCacheWizard, but given that the other 3 usages of this library work I think this is pretty safe.

Reply:
I agree. Thanks for the heads up!